go/bind/keybase.go

```diff
@@ -4,6 +4,7 @@
 package keybase
 
 import (
+	"context"
```

Comment:
Intended.  The interface definitions are compatible, so I'm trying to use this one whenever possible.  It only matters in very rare situations as far as I've seen.